src/apiData/cleanApiCalls.js

```diff
@@ -1,13 +1,16 @@
 const getPokemonDetails = (data) => {
   const { abilities, height, name, moves, types, weight } = data
-  console.log(data);
   const capitalName = name.charAt(0).toUpperCase() + name.slice(1)
-  const typesInfo = types.map(type => type.type.name)
-  console.log(typesInfo);
+  const typesInfo = types.map(type => type.type.name).join(' | ')
```

Comment:
Good job cleaning up the data! I like the way you are joining the types, abilities, and moves!